packages/intent-filler/src/config/confirmation-policy.ts

```diff
@@ -0,0 +1,45 @@
+export class ConfirmationPolicy {
+	// Maps chainId -> amount threshold -> confirmation blocks
+	private policies: Map<number, Map<bigint, number>>
```

Comment:
This confirmation policy feels a bit too naive. I think we can make this a bit better by ensuring that fillers provide the minimum amount they can fill for the least confirmations and the maximum amount they can fill alongside the required confirmations. 

Then we can plot a line between these two points and calculate the confirmation blocks for any amount that falls in between the two points.